.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,15 +2,14 @@
 Thanks for submitting a PR, your contribution is really appreciated!
 
 Here is a quick checklist that should be present in PRs.
-(please delete this text from the final description, this is just a guideline)
```

Comment:
This might be a misunderstanding then - I've assumed the whole text should be deleted - or at least we should not end up with open TODOs always (since you cannot target both master/features).
Can we have the closing `-->` at the end maybe, so everything becomes a comment / invisble?

Reply:
Sure, updated!